File: selection.js
function selectionSort(a) {
    for (let i = 0; i < a.length - 1; i++) {
        let minIndex = i;
        for (let j = i + 1; j < a.length; j++) {
            if (a[j] < a[minIndex])
                minIndex = j;
        }
        let temp = a[i];
        a[i] = a[minIndex];
        a[minIndex] = temp;
    }
}

function printArr(a) {
    console.log(a.join(" "));
}

let a = [70, 15, 2, 51, 60];
console.log("Antes de ordenar:");
printArr(a);
selectionSort(a);
console.log("Después de ordenar:");
printArr(a);

File: selection.java
public class Main {
    public static void selectionSort(int[] a) {
        for (int i = 0; i < a.length - 1; i++) {
            int minIndex = i;
            for (int j = i + 1; j < a.length; j++) {
                if (a[j] < a[minIndex])
                    minIndex = j;
            }
            int temp = a[i];
            a[i] = a[minIndex];
            a[minIndex] = temp;
        }
    }

    public static void printArr(int[] a) {
        for (int x : a)
            System.out.print(x + " ");
        System.out.println();
    }

    public static void main(String[] args) {
        int[] a = {70, 15, 2, 51, 60};
        System.out.println("Antes de ordenar:");
        printArr(a);
        selectionSort(a);
        System.out.println("Después de ordenar:");
        printArr(a);
    }
}
